src/styling/reset/ocl_reset_css.ts

```diff
@@ -0,0 +1,10 @@
+export const oclResetCss = `
+.react-ocl-reset input.react-ocl, .react-ocl-reset button.react-ocl {
```

Comment:
The reset is targeted, so is there an advantage of it being optional?

Reply:
1. Being optional is an advantage (no need to import it if already use a reset stylesheet)
2. this reset stylesheet is plan to be use on sites which does not use one